docs/grid/custom_action.md

```diff
@@ -1,13 +1,13 @@
-Creating custom Action
+Creating custom Actions
```

Comment:
Same question about file header as in previous https://github.com/Sylius/Stack/pull/180#discussion_r1888383950